WordPress/gutenberg
# Match color of the focus template view with site dark material

## What?
One minor SCSS change to adapt the background color of the focused template view with the same color as the dark material within the Site Editor. cc @jameskoster 

## Why?
Matching the colors will help with transitioning into the different focused states within the Site Editor. 

## How?
Changing the currently used `$gray-800` to `gray-900`. 

## Testing Instructions
1. Open the Site Editor.
2. Click on Template Parts.
3. Click on one of the template parts (i.e. Header).
4. See background color of the template part focus view matching that of the rest of the Site Editor. 

## Screenshots or screencast 

Before: 
<img width="700" alt="CleanShot 2023-02-24 at 19 10 01" src="https://user-images.githubusercontent.com/1813435/221257848-d57f41ac-4dc4-4bb8-8ea3-117f8b13bb1e.png">

After: 

<img width="697" alt="CleanShot 2023-02-24 at 19 10 13" src="https://user-images.githubusercontent.com/1813435/221257180-cb9f4742-8c61-4827-ab68-49cd6a161dd4.png">
